editions/tw5.com/tiddlers/widgets/CheckboxWidget.tid

```diff
@@ -47,6 +49,16 @@ This example creates a checkbox that is checked if the field ''status'' is equal
 
 <<wikitext-example-without-html """<$checkbox field="status" checked="open" unchecked="closed" default="closed"> Is it open?</$checkbox><br>''status:'' {{!!status}}""">>
 
+!! Field List Mode
+
+Using the checkbox widget in field mode requires the ''fieldMode'' attribute to be "list", and the ''field'' attribute to specify the name of the field. The ''checked'' attribute specifies the value to be added to (or removed from) the list to correspond to its checked and unchecked states respectively. If ''checked'' is empty but ''unchecked'' is set (to a non-empty value), then the logic will be inverted so that the checkbox will be checked if the value is missing from the list, or unchecked if the value is present. Finally, if both ''checked'' and ''unchecked'' are present, the checkbox will work like a toggle, replacing the ''checked'' value with the ''unchecked'' value and vice-versa.
```

Comment:
In the very first sentence I assume you mean "Using the checkbox widget in ~~field~~ list mode"

Reply:
Yep. Will fix.